src/ui/common/src/components/workflows/RequireDagOrResult.tsx

```diff
@@ -0,0 +1,68 @@
+import { Alert, AlertTitle, CircularProgress } from '@mui/material';
```

Comment:
This wrapper adds necessary checkings on `dagWithLoadingStatus` and `dagResultsWithLoadingStatus`, and returns alerts / in-progress icon accordingly.